scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
Then let's not have latest at all. We don't really want real clusters using the "latest" tag and for development we can build locally.